pandas/tests/test_base.py

```diff
@@ -1102,3 +1136,40 @@ def test_categorial_datetimelike(self, method):
 
         result = method(i)[0]
         assert isinstance(result, Timestamp)
+
+    def test_iter_box(self):
```

Comment:
Is this just a new test for coverage purposes or related to any of the issues you addressed?

Reply:
a moved test